priv/cabbage/apps/itest/test/itest/deposits_test.exs

```diff
@@ -33,9 +35,11 @@ defmodule DepositsTests do
     initial_balance = Itest.Poller.root_chain_get_balance(alice_account)
 
     {:ok, receipt_hash} =
-      amount
-      |> Currency.to_wei()
-      |> Client.deposit(alice_account, Itest.PlasmaFramework.vault(Currency.ether()))
+      Reorg.execute_in_reorg(fn ->
+        amount
+        |> Currency.to_wei()
+        |> Client.deposit(alice_account, Itest.PlasmaFramework.vault(Currency.ether()))
+      end)
```

Comment:
after the anon. function, the `balance = Client.get_exact_balance(alice_account, expecting_amount)` would def. return the correct and LAST balance because the reorg is at this point finished, correct?

Reply:
not really, it takes some time for childchain to sync the latest data